offa.csl

```diff
@@ -0,0 +1,438 @@
+            <if match="none" is-uncertain-date="accessed">
+              <group>
+                <text term="accessed" prefix=" ("/>
+                <date form="numeric" variable="accessed" suffix=")"/>
+              </group>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="doi">
+    <choose>
+      <if match="any" variable="DOI">
+        <group>
+          <text value=". DOI: https://doi.org/"/>
+          <text variable="DOI"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="review">
+    <choose>
+      <if match="any" variable="reviewed-author">
+        <text term="reviewed-author" form="verb" prefix=", " suffix=" "/>
+        <names variable="reviewed-author">
+          <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with="."/>
+        </names>
+        <text variable="reviewed-title" prefix=", "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="encyclopedia-article">
+    <text variable="container-title"/>
+    <number prefix=" " variable="volume"/>
+    <choose>
+      <if match="none" is-uncertain-date="issued">
+        <date date-parts="year" form="text" variable="issued" prefix=" (" suffix=")"/>
+      </if>
+    </choose>
+    <text variable="page" prefix=" "/>
+    <group>
+      <choose>
+        <if match="any" variable="title">
+          <text term="sub verbo" form="short" prefix=" " suffix=" "/>
+          <text variable="title"/>
+        </if>
+      </choose>
+    </group>
+    <text variable="edition" prefix=" [" suffix="]"/>
+    <choose>
+      <if match="any" variable="author">
+        <group prefix=" (" suffix=")">
+          <names variable="author"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="webpage-date">
+    <choose>
+      <if type="webpage" match="any">
+        <choose>
+          <if match="none" is-uncertain-date="issued">
+            <date date-parts="year" form="text" variable="issued" prefix=", "/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-givenname="true" givenname-disambiguation-rule="all-names-with-initials" disambiguate-add-year-suffix="true">
+    <sort>
+      <key variable="issued"/>
+      <key macro="contributors-short-citation"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text macro="contributors-short-citation"/>
+          <text macro="date"/>
+        </group>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" line-spacing="1" entry-spacing="1">
+    <sort>
+      <key macro="contributors-short-biblio"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <group suffix=".">
+        <group display="left-margin" suffix=": ">
```

Comment:
using left margin doesn't seem right. The bibliography just seems to start with the short author and have a hanging indent

Reply:
Corrected. Also corrected in style journal-of-neolithic-archaeology, as the offa-style derives from the latter.